src/Updown.ts

```diff
@@ -52,7 +52,7 @@ export default class Updown {
 	 * @param token   The token for a check
 	 * @param [page]  Page number (results are paginated per 100)
 	 */
-	public async getDowntime(token : string, page : number = 1) {
+	public async getDowntime(token: string, page = 1) {
```

Comment:
> @woubuc:
> Function arguments should always require type notations, for clarity.

> @MichaelDeBoey:
> I haven't touched any code, just added formatting.
> 
> But I was planning on doing this in a following PR.

> @MichaelDeBoey:
> It looks like this is coming from the `@typescript-eslint/no-inferrable-types` rule

> @woubuc:
> If you look at the diff, this line had `page : number = 1` but was changed by your PR.

Like I said: leaving it like this will create an error,  because of the `@typescript-eslint/no-inferrable-types` rule.

Reply:
I checked out the docs on the `no-inferrable-types` rule. I agree with their reasoning behind it for code itself (obviously you don't need to type every little variable you crate), but I find that function signatures and class properties are more easily understandable if the type notation is explicit (because it's the same type everywhere).

The `no-inferrable-types` rule actually has the options `ignoreParameters` and `ignoreProperties` which can be set to true to allow this, so it should be an easy fix.

Reply:
Yeah was looking into this too, because I indeed find it too that the type should stay in this case.

Reply:
Can easily be fixed by
```js
'@typescript-eslint/no-inferrable-types': ['error', { ignoreParameters: true }],
```